core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -89,7 +90,7 @@ public void createTable(
       addTableMetadata(connection, namespace, table, metadata, true);
     } catch (SQLException e) {
       throw new ExecutionException(
-          "creating the table failed: " + getFullTableName(namespace, table), e);
+          "creating the table failed: " + rdbEngine.encloseFullTableName(namespace, table), e);
```

Comment:
Because `getFullTableName` is hardcoded to use the `.` separator between a namespace and a table name.

The exception message in SQLite implementation;

- was: "creating the table failed: a.b"
- expected: "creating the table failed: a_b"

while `a` is the namespace and `b` is the table name.

Also, we should not change the `getFullTableName` implementation because it is used to calculate the `metadata.full_table_name` field content.

Reply:
I think we should show a ScalarDB table name (i.e. `a.b`) in the exception message. In fact, we show ScalarDB table names in the other exception messages. Maybe the actual table name is included in the cause exception `e`, so users can know the actual table name from it?

Reply:
466e0ccbfe88e7790fdd2cf5babfc224c13af9e9